backend/src/schema/resolvers/reports.js

```diff
@@ -58,54 +59,68 @@ export default {
     },
   },
   Query: {
-    reports: async (_parent, _params, { driver }, _resolveInfo) => {
+    reports: async (_parent, params, context, _resolveInfo) => {
+      const { driver } = context
       const session = driver.session()
-      const res = await session.run(
-        `
+      let response
+      let orderByClause
+      switch (params.orderBy) {
+        case 'createdAt_asc':
+          orderByClause = 'ORDER BY report.createdAt ASC'
+          break
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-16T23:05:25Z" title="Thursday, October 17th 2019, 1:05:25 am +02:00">Oct 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -58,54 +59,68 @@ export default {
     },
   },
   Query: {
-    reports: async (_parent, _params, { driver }, _resolveInfo) => {
+    reports: async (_parent, params, context, _resolveInfo) => {
+      const { driver } = context
       const session = driver.session()
-      const res = await session.run(
-        `
+      let response
+      let orderByClause
+      switch (params.orderBy) {
+        // case 'createdAt_asc':
+        //   orderByClause = 'ORDER BY report.createdAt ASC'
+        //   break
```


I don't understand why this is commented out?
maybe because there is no UI to allow `moderators` to order how they want, but I am guessing.
If that is the case, I don't see the need to leave it commented out, since there is more than one UI... we have the playground, for example.

if there is no `orderBy` it just comes in whatever order the database happens to return it?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-17T08:11:48Z" title="Thursday, October 17th 2019, 10:11:48 am +02:00">Oct 17, 2019</time>_

---


Okay. @mattwr18 
I was uncertain how to handle this.

> if there is no orderBy it just comes in whatever order the database happens to return it?

Yes! That was the cause I’ve seen that no ordering was applied.